sql_generators/glean_usage/glean_app_ping_views.py

```diff
@@ -81,6 +81,12 @@ def generate_per_app(
             # generate a unioned schema that contains all fields of all ping tables across all channels
             unioned_schema = Schema.empty()
 
+            # normalized_app_id is not part of the underlying table the schemas are derived from,
+            # the field gets added as part of the view definition, so we have to add it manually to the schema
+            unioned_schema.schema["fields"] = [
+                {"name": "normalized_app_id", "mode": "NULLABLE", "type": "STRING"}
+            ]
```

Comment:
I think adding `normalized_app_id` to the schema needs to be done after the SQL select statement is generated (e.g. right before generating the `schema.yaml` file below), otherwise the generated SQL select statement will try to select a (nonexistent) `normalized_app_id` column from the underlying tables (and it would also result in a duplicate select result column name).

Also, adding a column description might be nice.